pootle/apps/pootle_project/models.py

```diff
@@ -181,10 +181,12 @@ class Project(models.Model, CachedTreeItem, ProjectURLMixin):
 
     treestyle_choices = (
         # TODO: check that the None is stored and handled correctly
-        ('auto', _('Automatic detection (slower)')),
+        ('auto', _('Automatic detection of gnu/non-gnu file layouts (slower)')),
         ('gnu', _('GNU style: files named by language code')),
         ('nongnu', _('Non-GNU: Each language in its own directory')),
-    )
+        ('none', _('Do not scan or create directories '
+                   '(Select this option if you wish to manage files '
```

Comment:
I wasn't mentioning to unconfigure it, but disabling it. The suggestion of making it work automatic was orthogonal; it requires less UI, it's the default and it's probably what everyone will want to use.